src/inputs/durationPicker/index.js

```diff
@@ -1 +1 @@
-export { default } from './durationPicker';
+export { default } from './styledDurationPicker';
```

Comment:
Curious as to why we are doing this?

Reply:
So, I needed to export the main component itself for some of the unit testing, so I could access state and instance methods ... I couldn't figure how to do that with the mounted and wrapped component.  Maybe there is a way?  Anyhow, so I exported both from the one file, and then  the `react-docgen` package (used to generate the "Component API" documentation) was NOT happy ... see failed validation build https://dev.azure.com/saddlebackchurch/b8a0d0bb-a4fe-48e3-9a60-2f1440145fc4/_build/results?buildId=20429.

So this is what I came up with to appease all of the things.  Kind of strange, but it works.  See my comments on 0a8ea90 where I tried to explain.

If there is a way to do all of my testing using the HOC-wrapped and mounted component, so we don't have to export the inner component, I'm willing to refactor a bit.  I might need a bit of help with that.

Lemme know your preferences.

Reply:
I am not sure I am understanding, `wrapper.find('FooComponent').instance()` doesn't work with the `withStyles` HOC?

Reply:
I can try it again.  Specifically I recall having trouble using `.setProps()` in the test cases that are testing the logic in `componentDidUpdate()` ... I was getting an error message with something to the effect of only being able to call that from the root node.

Reply:
@morethanfire Can we maybe spend a few minutes discussing this issue?
Basically, I want to arrive at a decision of whether this is acceptable as-is, or if I need to make an effort to refactor the unit tests to avoid having to do this.  It'd be nice to get this resolve today if possible, obviously given other priorities and workload.

Reply:
Dude, you were totally right.  All fixed up in 058389b.  Sorry 'bout that.  I've learned now.

Reply:
Oh, Nice! Thank you.